app/Helpers/database/game.php

```diff
@@ -548,17 +548,21 @@ function getGamesList(?int $consoleID, ?array &$dataOut, bool $officialFlag = fa
     return count($dataOut);
 }
 
-function getGamesListDataNamesOnly(int $consoleID, bool $officialFlag = false): array
+function getGamesListDataNamesOnly(int $consoleId, bool $officialFlag = false): array
 {
-    $retval = [];
-
-    $data = getGamesListData($consoleID, $officialFlag);
-
-    foreach ($data as $element) {
-        $retval[$element['ID']] = utf8_encode($element['Title']);
-    }
-
-    return $retval;
+    return Game::with('system')
+        ->join('Console', 'GameData.ConsoleID', '=', 'Console.ID')
```

Comment:
`with('system')` sounds like it's already doing the join? 🤔 
can this be omitted or is it required for something else?

Reply:
Safe to omit, removed in latest.